markhor_slam/launch/markhor_slam.launch

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0"?>
+
+<launch>
+    <arg name="geotiff_map_file_path" default="$(find markhor_slam)/maps" />
+
+    <node pkg="rviz" type="rviz" name="rviz" args="-d $(find markhor_slam)/rviz_cfg/mapping.rviz" />
+
+    <!-- TF remapping -->
```

Comment:
I think it would be reasonable to add information about the reason we add those "remap" (I find it weird that we call those remapping since we are not really using the [remap](wiki.ros.org/roslaunch/XML/remap) tag). 

I think a small README.md would be appreciated.